http_request_randomizer/requests/useragent/userAgent.py

```diff
@@ -20,28 +30,29 @@ def load_user_agents(self, useragentsfile):
         return useragents
 
     def get_random_user_agent(self):
-        """
-        useragents : string array of different user agents
-        :param useragents:
-        :return random agent:
-        """
-        user_agent = random.choice(self.useragents)
-        return user_agent.decode('utf-8')
+        if self.agent_file:
+            user_agent = random.choice(self.useragents)
+            return user_agent.decode('utf-8')
+        else:
+            return self.fakeuseragent.random
 
     def get_first_user_agent(self):
-        return self.useragents[0].decode('utf-8')
+        return self.useragents[0].decode('utf-8') if self.agent_file else None
```

Comment:
Shall we print a WARNING message when this method is called with fakeuseragent?

Reply:
Yes we should, but o just noticed the logging doesn't work in this class. Can you help me why?

Reply:
Get rid of the logger.setLevel part - this is done in the caller/main class